scenes/quests/story_quests/template/template_player_components/template_player.tres

```diff
@@ -1,9 +1,9 @@
-[gd_resource type="SpriteFrames" load_steps=19 format=3 uid="uid://vwf8e1v8brdp"]
+[gd_resource type="SpriteFrames" load_steps=17 format=3 uid="uid://vwf8e1v8brdp"]
 
 [ext_resource type="Texture2D" uid="uid://c0itxt8w3yml0" path="res://scenes/quests/story_quests/template/template_player_components/template_player_attack_01.png" id="1_m5whc"]
-[ext_resource type="Texture2D" uid="uid://b0kexdddbmi7n" path="res://scenes/quests/story_quests/template/template_player_components/template_player_idle.png" id="2_bi3qj"]
 [ext_resource type="Texture2D" uid="uid://dly3ikrx05f0x" path="res://scenes/quests/story_quests/template/template_player_components/template_player_defeated.png" id="2_qxir4"]
-[ext_resource type="Texture2D" uid="uid://bnb6fdjs7xeaa" path="res://scenes/quests/story_quests/template/template_player_components/template_player_walk.png" id="3_qxir4"]
+[ext_resource type="Texture2D" uid="uid://ujemrbowjxkn" path="res://scenes/quests/story_quests/luca/luca_components/template_Luca_idle.png" id="3_bi3qj"]
+[ext_resource type="Texture2D" uid="uid://ro2oxwv0yc32" path="res://scenes/quests/story_quests/luca/luca_components/template_Luca_walk.png" id="4_qxir4"]
```

Comment:
You have modified the `template_player.tres` file in the `template` directory. You should instead modify `luca_player.tres` in your `luca` directory, and revert these changes.